skipper/privileged_user.go

```diff
@@ -0,0 +1,70 @@
+package skipper
+
+import (
+	"github.com/minio/pkg/wildcard"
```

Comment:
Any reason to use minios implementation specifically? I was very confused why we suddenly depend on minio :sweat_smile: 

Random idea. Maybe https://pkg.go.dev/path/filepath#Match might actually do the trick?

Reply:
`filepath.Match` has some specialities for path handling. They most likely don't matter here but i'd rather be safe.

There's also a fork of the Minio implementation. Which does not pull in all of minio. But it might not get the same love.

Reply:
Fair enough. I'm fine with using the minio implementation, I just seemed odd at first